client/src/App.js

```diff
@@ -1,9 +1,41 @@
+import { useEffect, useState } from 'react';
 import Ohm from './pages/Ohm';
+import ExercisesCategory from './components/ExercisesCategory';
+import BodyParts from './pages/BodyParts';
+import { Route, Switch } from 'react-router-dom';
 
 function App() {
+  const [exercises, setExercises] = useState([]);
+  const [timerExpired, setTimerExpired] = useState(false);
+
+  //Verknüpfung back and front
+  useEffect(() => {
+    fetch('http://localhost:4000/exercises')
+      .then((res) => res.json())
+      .then((data) => {
+        console.log(data);
+        setExercises(data);
+      })
+      .catch((err) => console.log('Error 400'));
```

Comment:
I think you will not always have a 400 error. Just log the error message provided 

```suggestion
      .catch((err) => console.log(err.message));
```